com.unity.render-pipelines.core/Editor/Lighting/SRPLensFlareElementEditor.cs

```diff
@@ -0,0 +1,490 @@
+                    SRPLensFlareBlendMode blendModeValue = (UnityEngine.SRPLensFlareBlendMode)blendModeProp.enumValueIndex;
+                    if ((newBlendMode = ((SRPLensFlareBlendMode)(EditorGUI.EnumPopup(rect, Styles.blendMode, blendModeValue)))) != blendModeValue)
+                        blendModeProp.enumValueIndex = (int)newBlendMode;
+                }
+
+                rect = GetNextRect();
+                EditorGUI.TextArea(rect, "Transform", EditorStyles.boldLabel);
+                {
+                    rect = GetNextRect();
+                    if ((tmpVec2 = EditorGUI.Vector2Field(rect, Styles.positionOffset, positionOffsetProp.vector2Value)) != positionOffsetProp.vector2Value)
+                        positionOffsetProp.vector2Value = tmpVec2;
+                    rect = GetNextRect();
+                    if ((tmpBool = EditorGUI.Toggle(rect, Styles.autoRotate, autoRotateProp.boolValue)) != autoRotateProp.boolValue)
+                        autoRotateProp.boolValue = tmpBool;
+                    rect = GetNextRect();
+                    if ((tmp = EditorGUI.FloatField(rect, Styles.rotation, rotationProp.floatValue)) != rotationProp.floatValue)
+                        rotationProp.floatValue = tmp;
+
+                    if (!preserveAspectRatioProp.boolValue)
+                    {
+                        rect = GetNextRect();
+                        if ((tmpVec2 = EditorGUI.Vector2Field(rect, Styles.sizeXY, sizeXYProp.vector2Value)) != sizeXYProp.vector2Value)
+                            sizeXYProp.vector2Value = new Vector2(Mathf.Max(tmpVec2.x, 1e-6f), Mathf.Max(tmpVec2.y, 1e-6f));
+                    }
+
+                    rect = GetNextRect();
+                    if ((tmp = EditorGUI.FloatField(rect, Styles.uniformScale, uniformScaleProp.floatValue)) != uniformScaleProp.floatValue)
+                        uniformScaleProp.floatValue = Mathf.Max(tmp, 0.0f);
+                }
+
+                rect = GetNextRect();
+                EditorGUI.TextArea(rect, "Axis Transform", EditorStyles.boldLabel);
+                {
+                    rect = GetNextRect();
+                    if ((tmp = EditorGUI.FloatField(rect, Styles.position, positionProp.floatValue)) != positionProp.floatValue)
+                        positionProp.floatValue = tmp;
+                    rect = GetNextRect();
+                    if ((tmp = EditorGUI.FloatField(rect, Styles.angularOffset, angularOffsetProp.floatValue)) != angularOffsetProp.floatValue)
+                        angularOffsetProp.floatValue = tmp;
+                    rect = GetNextRect();
+                    if ((tmpVec2 = EditorGUI.Vector2Field(rect, Styles.translationScale, translationScaleProp.vector2Value)) != translationScaleProp.vector2Value)
+                        translationScaleProp.vector2Value = tmpVec2;
+                }
+
+                rect = GetNextRect();
+                EditorGUI.TextArea(rect, "Radial Distortion", EditorStyles.boldLabel);
```

Comment:
label should be in a style class to not search them in the logic and it should be localizable.
Please use either **L18n** or **EditorGUIUtility.TrTextContent**